src/market.rs

```diff
@@ -72,8 +96,10 @@ pub type OrdersApiResponse = ApiResponse<OrdersPayload>;
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ItemsItem {
     pub id: String,
-    pub url_name: String,
-    pub item_name: String,
+    #[serde(rename = "urlName")]
```

Comment:
Are you sure that there must be `urlName`? Not `url_name`?

Reply:
I thought you knew that `serde` by default use `camelCase`. 
If warface.market provide `url_name` you should fix this